scripts/commons.sh

```diff
@@ -2,11 +2,11 @@
 
 set -euo pipefail
 
-
-BUILD_DIR="./bin"
-SOURCE_DIR="./src"
-PROVIDER_NAME_FILE="./PROVIDER_NAME.txt"
-PROVIDER_VERSION_FILE="./PROVIDER_VERSION.txt"
+SCRIPTS_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )
```

Comment:
Can be simplified to the following:
```suggestion
SCRIPTS_DIR=$(dirname $0)
```